sample_code/ep028/rev02/t.py

```diff
@@ -5,7 +5,7 @@ def c_main(stdscr: 'curses._CursesWindow') -> int:
     while True:
         # rendering phase
         ...
-        char = stdscr.get_wch()
+        stdscr.get_wch()
```

Comment:
Do you want me to also revert the unused vars like this `char` assignment and the `args` for ArgParse?

Reply:
yeah, they were placeholders imaginging you'd use the arguments / characters afterwards

another idea might be to use `per-file-ignore`s to exclude the particular rules in particular files

Reply:
I saw that option, but which file should we use for the ignore metadata `setup.cfg`, `tox.ini` or `.flake8`? I am not sure.

Reply:
I usually prefer setup.cfg unless tox.ini exists

Reply:
Ok, we have the setup.cfg with flake8 `per-file-ignore`.